web/webkit/src/main/scala/net/liftweb/http/Req.scala

```diff
@@ -284,14 +284,14 @@ FileParamHolder(name, mimeType, fileName)
  * @param localFile The local copy of the uploaded file
  */
 class OnDiskFileParamHolder(override val name: String, override val mimeType: String,
-                            override val fileName: String, val localFile: File) extends
+                            override val fileName: String, val localFile: Path) extends
```

Comment:
Yeah not sure either. Will keep it in mind in case anything around this comes up in the future.